tests/functional/tests/conftest.py

```diff
@@ -37,3 +38,18 @@ def pyocf_ctx_log_buffer():
     yield logger
     for cache in c.caches:
         cache.stop()
+
+
+@pytest.fixture()
+def start_cache_with_core():
+    def start_with_config(cache_size: Size, core_size: Size, **config):
```

Comment:
Is this a fixture that returns a function to call when wanting to start cache/core pair? Consider putting it in some test utils (I believe there is already a tests.utils module), I'm not sure writing this as a fixture is a good pattern.

Reply:
done